isis/src/base/objs/Cube/Cube.cpp

```diff
@@ -759,10 +809,14 @@ namespace Isis {
       m_ioHandler = new CubeBsqHandler(dataFile(), m_virtualBandList,
           realDataFileLabel(), true);
     }
-    else {
+    else if (m_format == Tile) {
       m_ioHandler = new CubeTileHandler(dataFile(), m_virtualBandList,
           realDataFileLabel(), true);
     }
+    else if (m_format == GTiff) {
+      QString datafile = realDataFileName().expanded();
+      m_ioHandler = new GdalIoHandler(datafile, m_virtualBandList);
+    }
```

Comment:
same here same here, if label type == gdal then dn format I think should also be gdal.

Reply:
Similar to previous response. I don't know if we need to explicitly define that we have a `Gdal` label